pdsfile/pds4file/rules/cassini_iss_fring_mosaics_rsfrench2025.py

```
@@ -0,0 +1,328 @@
+    (r'bundles/cassini_iss_fring_mosaics_rsfrench2025/cassini_iss_fring_mosaics_rsfrench2025[^/]*/browse_reproj_img/(iss|iosic)_.*/.*_browse_reproj_img_small\.png',              0, ('Cassini ISS', 180, 'coiss_f_ring_browse_reproj_img_small', 'Browse Reprojected Image (small)', True)),
+    (r'bundles/cassini_iss_fring_mosaics_rsfrench2025/cassini_iss_fring_mosaics_rsfrench2025[^/]*/browse_reproj_img/(iss|iosic)_.*/.*_browse_reproj_img_med\.png',              0, ('Cassini ISS', 190, 'coiss_f_ring_browse_reproj_img_med', 'Browse Reprojected Image (medium)', True)),
+    (r'bundles/cassini_iss_fring_mosaics_rsfrench2025/cassini_iss_fring_mosaics_rsfrench2025[^/]*/browse_reproj_img/(iss|iosic)_.*/.*_browse_reproj_img_full\.png',              0, ('Cassini ISS', 200, 'coiss_f_ring_browse_reproj_img_full', 'Browse Reprojected Image (full)', True)),
+])
+
+##########################################################################################
+# OPUS_PRODUCTS
+##########################################################################################
+
+opus_products = translator.TranslatorByRegex([
+    (r'bundles/(cassini_iss_fring_mosaics_rsfrench2025/cassini_iss_fring_mosaics_rsfrench2025[^/]*)/(data|browse)_mosaic.*/((iss|iosic)_[a-zA-Z0-9_]*)/(iss|iosic)_[a-zA-Z0-9_]*_mosaic.*\.[a-z]{3,4}', 0,
+    [
```

Flake8 (7.3.0)

[error] continuation line under-indented for visual indent

(E128)